docs/src/app/components/pages/components/dialog.jsx

```diff
@@ -72,6 +76,12 @@ export default class DialogPage extends React.Component {
             desc: 'Set to true to have the dialog automatically open on mount.',
```

Comment:
Is there a convention on this? Can't find any existing deprecations with warnings.